stdlib/http/client.pyi

```diff
@@ -169,6 +169,9 @@ class HTTPConnection:
     ) -> None: ...
     def getresponse(self) -> HTTPResponse: ...
     def set_debuglevel(self, level: int) -> None: ...
+    if sys.version_info >= (3, 12):
+        def get_proxy_response_headers(self) -> HTTPMessage | None: ...
```

Comment:
The docstring says "returns a dictionary" but that appears to be a lie, this type is right